src/inc/db.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
db.php should not exist in the repository as it should be created from db.template.php either manually or by the installation process.